src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -94,6 +97,17 @@ class PluginMultipleReference:
     item_id_list = attr.ib(default=attr.Factory(list))
 
 
+@attr.s(frozen=True, slots=True)
+class PluginTableColumnForTracerReference:
```

Comment:
For your consideration: `TracerReferencePluginTableColumn` (same for the other class).

Reply:
- [ ]  rename classes;

Reply:
BTW, I did prefix with `PluginTableColumn` to help with discoverability, so column types and table are listed near to each other in alphabetical order.

Reply:
OK -- I assume you just commenting this to convey the rationale, but still intend to rename them?

Reply:
Yes, I did.